Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_-_Search_Endpoints_By_Hash.yml

```diff
@@ -936,3 +936,4 @@ tests:
 contentitemexportablefields:
   contentitemfields: {}
 system: true
+
```

Comment:
is there a chang in the playbook? if so - no change made
if no - please remove the extra line (avoid the change)

Reply:
NP make sense